### PR TITLE
refactor(config): use templates for Yale locks

### LINKED ISSUE #1882: Yale lock YRD226, but it has a new product ID that doesn't exist in the z-wave DB
I have a Yale lock `YRD226-ZW2-619`. The `YRD226` is in the [z-wave product database](https://products.z-wavealliance.org/products/1878?selectedFrequencyId=2) with a product type of `0x8002` and a product ID of `0x0600`. However, my `YRD226` is reporting a product ID of `0x4600`, which means it shows up in `zwavejs2mqtt` as unknown. The problem is I cannot find this product ID in the z-wave products database. Is it normal for manufacturers to update the product ID but it doesn't show up in the z-wave products DB? Should I add this product ID to the existing YRD226 config file?

Here is the zwavejs log showing the product ID:

```
10:19:12.270 CNTRLR   [Node 005] [+] [Manufacturer Specific] manufacturerId: 297        [Endpoint 0]
10:19:12.271 CNTRLR   [Node 005] [+] [Manufacturer Specific] productType: 32770         [Endpoint 0]
10:19:12.271 CNTRLR   [Node 005] [+] [Manufacturer Specific] productId: 17920           [Endpoint 0]
10:19:12.272 DRIVER « [Node 005] [REQ] [ApplicationCommand]
                      └─[SecurityCCCommandEncapsulation]
                        │ sequenced: false
                        └─[ManufacturerSpecificCCReport]
                            manufacturer id: 0x0129
                            product type:    0x8002
                            product id:      0x4600
10:19:12.273 CNTRLR « [Node 005] received response for manufacturer information:
                        manufacturer: ASSA ABLOY (0x0129)
                        product type: 0x8002
                        product id:   0x4600
```

### PR DESCRIPTION
**Pull request by blhoward2**
Adds templates to Yale locks. This batch took forever as there were a lot of errors in the files. I did the best I could going back to the manual for each one, but we'll likely have to fix some things as users test. A few of these have no manuals that I could find.

Closes #1882 as part of this push.

### CODE REVIEW

**Comment by blhoward2** [comment]
@zwave-js-bot fix lint

**Comment by blhoward2** [comment]
@zwave-js-bot fix lint

**Comment by blhoward2** [comment]
@zwave-js-bot fix lint

**Comment by AlCalzone** [comment]
> 🐌 Please wait for the lint check to complete, then try again.

I think this message is suboptimal. We shouldn't offer the lint fix if there is nothing to fix automatically.

**Comment by blhoward2** [comment]
Yes, it took me a few tries to realize it couldn't fix it.

File: packages/config/config/devices/0x0129/templates/yale_template.json [hunk 1]
```diff
@@ -0,0 +1,333 @@
...
+		"minValue": 1,
+		"maxValue": 6,
+		"defaultValue": 2,
+		"unsigned": true,
+		"readOnly": false,
+		"writeOnly": false,
+		"allowManualEntry": false,
+		"options": [
+			{
+				"label": "Korean language",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0129/templates/yale_template.json:181]: Why not just `"Korean"`?

File: packages/config/config/devices/0x0129/templates/yale_template.json [hunk 2]
```diff
@@ -0,0 +1,333 @@
...
+		"label": "Door Propped Alarm"
+	},
+	"enable_dps_alarm": {
+		"$import": "../../templates/master_template.json#base_enable_disable",
+		"label": "Door Propped Alarm",
+		"defaultValue": 1
+	},
+	"deadbolt_installed_state": {
+		"$import": "../../templates/master_template.json#base_enable_disable_255",
+		"label": "Deadbolt Status",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0129/templates/yale_template.json:274]: Is this a readonly parameter?

File: packages/config/config/devices/templates/master_template.json [hunk 3]
```diff
@@ -78,6 +100,26 @@
 		"writeOnly": false,
 		"allowManualEntry": true
 	},
+	"base_0-180_nounit": {
+		"valueSize": 1,
+		"minValue": 0,
+		"maxValue": 180,
+		"defaultValue": 30,
```
Comment (AlCalzone) [inline packages/config/config/devices/templates/master_template.json]: This default value seems a bit arbitrary for a master template IMO.
I'm sure there are many other devices where a parameter has a range of 0-180 with a different default value.

File: packages/config/config/devices/templates/master_template.json [hunk 4]
```diff
@@ -78,6 +100,26 @@
...
+		"unsigned": true,
+		"readOnly": false,
+		"writeOnly": false,
+		"allowManualEntry": true
+	},
+	"base_0-127_nounit": {
+		"valueSize": 1,
+		"minValue": 0,
+		"maxValue": 127,
+		"defaultValue": 5,
```
Comment (AlCalzone) [inline packages/config/config/devices/templates/master_template.json]: dito

**Review by AlCalzone — CHANGES_REQUESTED** [review]
A few minor things

**Review by AlCalzone — APPROVED** [review]

**Pull request merged**